actions/issues.go

```diff
@@ -149,3 +227,42 @@ func requestParamToQueryFilter(query, paramValue, paramName *string) {
 		}
 	}
 }
+
+func preCacheIssues(whereClause string, params buffalo.ParamValues, page string) {
+	tx, err := models.DB.NewTransaction()
+	defer tx.Close()
+
+	if err != nil {
+		fmt.Println(errors.New("no transaction found"))
+	}
+
+	cacheConn := cache.CachePool.Get()
+	defer cacheConn.Close()
+
+	issues := &models.Issues{}
+	nextPage, _ := strconv.Atoi(page)
+	nextPage++
+	nextPageStr := strconv.Itoa(nextPage)
+	nextParams := params.(url.Values)
+	nextParams.Set("page", nextPageStr)
+	nextQ := tx.PaginateFromParams(nextParams).Eager()
+
+	nextCacheKey := "issues:" + whereClause + "and page=" + nextPageStr
+
+	ok, _ := cache.Exists(&cacheConn, nextCacheKey)
+
+	if ok == false {
+		if err := nextQ.Where(whereClause).All(issues); err != nil {
+			fmt.Println(errors.WithMessage(err, "DB Operation falied"))
+			fmt.Println(nextQ)
+			return
+		}
+
+		jsonIssues, _ := json.Marshal(issues)
+		_, err := cache.SetEx(&cacheConn, nextCacheKey, 600, string(jsonIssues))
```

Comment:
could you also add the count of the issues to cache?

Reply:
Do you mean the total count of issues for the query not paged result (issues-count) ?

Reply:
Sorry my suggestion doesn't make sense, I'll remove it. My bad